cobaya/likelihoods/planck_2018_lowl/TT_native.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+from cobaya.likelihoods.base_classes import InstallableLikelihood
+
+class TT_native(InstallableLikelihood):
+    """
+    Python translation of the Planck 2018 Gibbs TT likelihood (python Eirik Gjerløw, Feb 2023)
+    See https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/CMB_spectrum_%26_Likelihood_Code
+
+    """
+
+    install_options = {"github_repository": "CobayaSampler/planck_native_data",
+                       "github_release": "v1",
+                       "asset": "planck_2018_lowT.zip",
+                       "directory": "planck_2018_lowT_native"}
+
+    type = "CMB"
+    aliases = ["lowT"]
+
+    _lmin = 2
+    _lmax = 29 # Could in principle be extended to 200
+    _delta_l = 1000
+    _nbins = 0
+
+
+    _data = {}
+    _data['cov'] = np.zeros((251, 251))
+    _data['cl2x'] = np.zeros((1000, 251, 3))
+    _data['mu'] = np.zeros(251)
+    _data['mu_sigma'] = np.zeros(251)
+
```

Comment:
Looks like these could just be variables rather than elements of a dictionary (better for code validators)

Reply:
The 2:lmax range could also be set up in initialize rather than having explicit cuts later at likelihood run time.

Reply:
(I'm not  very sure what cl2x is - some comments might be helpful!)